plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseConsumerSettings.java

```diff
@@ -48,8 +49,11 @@ public class DatabaseConsumerSettings implements IDataTransferSettings {
 
     private static final Log log = Log.getLog(DatabaseConsumerSettings.class);
 
+    @Deprecated
     private String containerNodePath;
+    @Deprecated
```

Comment:
need a comment that these fields exist for backwards compatibility, otherwise someone will delete them in the future